target_api/client.py

```diff
@@ -82,8 +84,20 @@ def response_error_message(self, response: requests.Response) -> str:
         return f"Status code: {response.status_code} with {response.reason} for path: {response.request.url} {response_text}"
     
     def curlify_on_error(self, response):
-        curl = to_curl(response.request)
-        return curl
+        command = "curl -X {method} -H {headers} -d '{data}' '{uri}'"
+        method = response.request.method
+        uri = response.request.url
+        data = response.request.body
+
+        headers = []
+        for k, v in response.request.headers.items():
+            # Mask the Authorization header
+            if k.lower() in ["authorization", "x-api-key", self._config.get("api_key_header")]:
```

Comment:
Won't this fail if `self._config.get("api_key_header")` is not all lower case?

Reply:
good catch, thanks